BUILDING.txt

```diff
@@ -540,6 +564,11 @@ configure the bit-ness of the build, and set several optional components.
 Several tests require that the user must have the Create Symbolic Links
 privilege.
 
+To simplify the installation of Boost, Protocol buffers, OpenSSL and Zlib dependencies we can use
+vcpkg (https://github.com/Microsoft/vcpkg.git). Upon cloning the vcpkg repo, checkout the commit
+7ffa425e1db8b0c3edf9c50f2f3a0f25a324541d to get the required versions of the dependencies
```

Comment:
Why not a specific branch or latest?
It would also be good to add the command.

Reply:
> Why not a specific branch or latest?
I couldn't find a branch/release in vcpkg where all these dependencies are at the required versions.

> It would also be good to add the command.
Done.